package/avahi/avahi.mk

```diff
@@ -204,6 +204,13 @@ endef
 
 AVAHI_POST_INSTALL_TARGET_HOOKS += AVAHI_KNOT_SERVICE
 
+define AVAHI_DISABLE_IPV6
```

Comment:
You are going to disable IPv6 always doing that. 
Here you should create a flag to enable/disable IPv6 in ` package/avahi/Config.in` file. As the default is to enable ipv6 you should also add the flag disabling it on `configs/knot_gateway_defconfig`
Like this:
`ifeq ($(BR2_PACKAGE_AVAHI_DISABLE_IPV6),y)`
`... your changes`
`endif`